model/task_queue_service.go

```diff
@@ -15,7 +15,12 @@ import (
 	"github.com/pkg/errors"
 )
 
-// TODO rename this interface?
+// type TaskQueueItemDispatcher interface {
+// 	FindNextTask(string, TaskSpec) (*TaskQueueItem, error)
+// 	Refresh(string) error
+// 	RefreshFindNextTask(string, TaskSpec) (*TaskQueueItem, error)
+// }
```

Comment:
why is this commented out? I think we should probably just remove this as its a copy of the taskQueueService

Reply:
As agreed in person: we've renamed the interface `TaskQueueService` to `TaskQueueItemDispatcher`.